docs/guidelines/platforms/ios/guideline_understandable_ios.md

```diff
@@ -177,6 +228,91 @@ usernameHolderView.shouldGroupAccessibilityChildren = true
 
 A user enters some data into a user interface element without knowing what the end result is and/or triggering an unannounced change of context in the process.
 
+### Error Suggestion (WCAG 3.3.3 - Level AA)
+
+To provide the best user experience, the application should suggest solutions for input errors when they are detected.
+
+> This technique covers point *3.3.3 Error Suggestion - Level AA of the WCAG standard.*
+
+#### ✅ Success technique(s)
+
+When an error is detected, the application should provide a suggestion for the user on how to correct it. This can be done by providing a hint or a description of the error, or providing a suggestion on how to correct it.
```

Comment:
```suggestion
When an error is detected, the application should provide a suggestion for the user on how to correct it. This can be done by providing a hint or a description of the error or providing a suggestion on how to correct it.
```